src/ngraph_rewrite_pass.cc

```diff
@@ -102,13 +106,33 @@ class NGraphRewritePass : public GraphOptimizationPass {
   static std::string GraphFilenamePrefix(std::string kind, int idx) {
     std::stringstream ss;
     ss << kind << "_" << std::setfill('0') << std::setw(4) << idx;
+#ifdef NGRAPH_DISTRIBUTED
```

Comment:
Consider a function that can be used instead of the code repetition below.

Reply:
Used the distributed class defined in nGraph core